var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -17,7 +17,16 @@ class PyPennylaneLightning(CMakePackage, PythonExtension):
     maintainers("mlxd", "AmintorDusko")
 
     version("master", branch="master")
-    version("0.29.0", sha256="da9912f0286d1a54051cc19cf8bdbdcd732795636274c95f376db72a88e52d85")
+    version(
```

Comment:
Do we need to have changes in the Py-PennyLane-Lightning file for this? I think the new package should be pretty orthogonal to this one